benchmarks/rayleigh_taylor_instability/rayleigh_taylor_instability.prm

```diff
@@ -0,0 +1,74 @@
+set Additional shared libraries  = ./librayleigh_taylor_instability.so 
```

Comment:
What is the difference between `blank.prm` and `rayleigh_taylor_instability.prm`? Do you need both? Please add a comment at the top to explain.

Reply:
It is now explained in README.md